README.md

```diff
@@ -103,6 +103,12 @@ packed into the `plugin_instance` dimension using the following syntax:
 ```
 plugin_instance:value[dim1=value1,dim2=value2,...]
 ```
+The dimensions can then be used as filters. Using the same examples as above, such filters would look like:
```

Comment:
I don't think this paragraph belongs here. It's important to keep the following paragraph ("Because CollectD ...") close to the information above lines 100-105.

I'm not entirely convinced this information is useful here, as it relates more to the monitoring system those metrics are reported into. And in that context, it's understood that dimensions can be used as filters on graphs/detectors.